src/test/java/com/cegeka/academy/service/ChallengeServiceTest.java

```diff
@@ -87,7 +88,7 @@ public void init(){
 
         challenge = new Challenge();
 
-        challenge.setCreator(user);
+        challenge.setCreator(userRepository.findAll().get(0));
```

Comment:
De ce ai schimbat aici?

Reply:
Am schimbat deoarece obiectul user are id-ul null. Id-ul pentru el se seteaza cand faci save.

Reply:
Nop. Se autocompleteaza cand dai save. Am intrat special in debugger sa verific asta.

Reply:
da, am verificat si ai dreptate